### ISSUE TITLE
[FEATURE] "Multitasking" insbesondere Timer

### ISSUE THREAD

**Issue by alexhell** [jlawyerorg/j-lawyer-org#2364]
**Beschreibung**

Grds. kann man nur eine Sache zur Zeit machen. Das behindert etwas den Workflow. Wenn man z.B. zwischen zwei Akten springen muss etc.

Das mache ich im Moment sequenziell.

Ein solches Vorgehen scheitert aber, wenn man den Timer laufen lässt und dann in JL arbeiten möchte. Das geht dann einfach nicht mehr. 

Wenn ich z.B. während einer telefonischen Beratung schnell mal die Akte einsehen oder Adressdaten aktualisieren möchte, ist dies unmöglich.

**Comment by j-dimension** [comment]
Die grundlegende Thematik kann ich nachvollziehen, insofern ist dieses Ticket m.E. ein Duplikat von #402 

Das mit den Timern verstehe ich nicht ganz  - geht es um Zeiterfassung? Diese ist ja unabhängig davon, wo man sich in der Anwendung befindet, für beliebige Zeiterfassungsprojekte zu starten und zu stoppen (über das Icon in der Fußzeile).

**Comment by alexhell** [comment]
ja, es geht um die Zeiterfassung.  Ich hatte sie für eine Telefonberatng über die Kopfzeile gestartet und wollte dann in der Akte etwas machen. Das ging dann nicht mehr.

**Comment by j-dimension** [comment]
Du kannst jederzeit und aus beliebiger Perspektive / Dialog für eine beliebige Akte über die Fußzeile (Stoppuhrsymbol) eine Zeiterfassung starten und den Dialog mit laufender Zeit schließen - sie läuft im Hintergrund weiter. 
Bei laufender Zeiterfassung gibt es eine blinkene Ausgabe in der Fußzeile.

Vergisst man eine Zeit zu stoppen, wird man spätestens bei Schließen der Anwendung darauf hingewiesen.

Hilft Dir das weiter?

**Comment by stm9x9** [comment]
aber man kann dann die unterbrochene Aufgabe nicht einfach wieder aufnehmen, oder? Also in der Zeiterfassung.

**Comment by j-dimension** [comment]
Die letzte Aktivität steht ganz oben, auch mit der zuletzt angegebenen Tätigkeitsbeschreibung. Man klickt also einfach wieder auf "Start".
